LearningHub.Nhs.WebUI/Controllers/AccountController.cs

```diff
@@ -1265,10 +1265,11 @@ private async Task<AccountCreationConfirmation> GetAccountConfirmationDetails(Ac
             var employer = await this.locationService.GetByIdAsync(int.TryParse(accountCreationViewModel.LocationId, out int primaryEmploymentId) ? primaryEmploymentId : 0);
             var region = await this.regionService.GetAllAsync();
             var specialty = await this.specialtyService.GetSpecialtiesAsync();
-            var role = await this.jobRoleService.GetPagedFilteredAsync(accountCreationViewModel.CurrentRoleName, accountCreationViewModel.CurrentPageIndex, UserRegistrationContentPageSize);
```

Comment:
Guessing we've deliberately switched the method called?

Reply:
Yes, paginated result blocked from fetching the current user role.